.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "strict": 0
```

Comment:
Our default file [is much bigger](https://github.com/codeclimate/codeclimate/blob/master/config/eslint/.eslintrc). What made you go with this?